apiserver/facades/client/spaces/spaces.go

```diff
@@ -249,3 +312,16 @@ func (api *API) ReloadSpaces() error {
 	}
 	return errors.Trace(api.backing.ReloadSpaces(env))
 }
+
+// supportsSpaces checks if the environment implements NetworkingEnviron
+// and also if it supports spaces.
+func (api *API) supportsSpaces() error {
```

Comment:
Small nitpick: `supportsSpaces` reads like a function that one would expect to return a bool value. Perhaps something like `checkSupportForSpaces` or `ensureSpacesSupport` would read better.

Feel free to ignore this comment.

Reply:
changed to checkSupportsSpaces().